internal/services/applications/application_resource_test.go

```diff
@@ -33,6 +33,7 @@ func TestAccApplication_basic(t *testing.T) {
 				check.That(data.ResourceName).Key("application_id").Exists(),
 				check.That(data.ResourceName).Key("client_id").Exists(),
 				check.That(data.ResourceName).Key("object_id").Exists(),
+				check.That(data.ResourceName).Key("app_password").DoesNotExist(),
```

Comment:
Should be `password`

```suggestion
				check.That(data.ResourceName).Key("password").DoesNotExist(),
```